step-ca/certificate-authority-server-production.mdx

```diff
@@ -600,6 +602,85 @@ By default, `step-ca` uses passive revocation.
+  However, many of these tools rely on `ssh`, which comes with its own bootstrapping challenges.
+  See our blog post [If you’re not using SSH certificates you’re doing SSH wrong](https://smallstep.com/blog/use-ssh-certificates/) for more. 
+- For MDM environments,
+  you can leverage the MDM channel's TLS connection,
+  which is established upon MDM enrollment.
+
+#### Installing a CA certificate
+
+Your CA certificate may need to be installed and trusted into the target operating system's certificate trust store.
+The command for doing this various by operating system, distribution, and the target application.
+Confusingly, there may be several trust store libraries and locations on a given system.
+
+#### Example: Linux Trust Anchors
+
+While many popular Linux distributions have standardized around the [`ca-certificates`](https://src.fedoraproject.org/rpms/ca-certificates) package to manage their internal trust store,
+they don't all use the same paths for the trust store itself.
+
+Unfortunately, there is no single place on a Linux system where you can reliably mount this volume,
+because **applications use several methods to access system CA certificates:**
+
+- OpenSSL-based applications and many Linux distributions use the `/etc/ssl/certs` directory as the system trust store.
+- NSS uses a hard-coded list of trusted CA certificates inside the `libnssckbi.so` library,
+  and it stores user-supplied certificates in SQL databases inside `$HOME/.pki/nssdb` or `/etc/pki/nssdb`.
+- Java applications use a special Java Keystore file and the [`keytool`](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) command to manage it.
+- Some applications (like `step`) don't trust *any* system trust store,
+  or they have their own independent trust store or need to be configured with a CA bundle filename.
+  Servers that need to validate client certificates generally won't trust the system trust store,
+  and will require a CA bundle be passed in as a config parameter.
+
+**For distros that use standard `ca-certificates` paths**
+
+- Debian
+- BusyBox
+- Ubuntu
+- Alpine
+
+The recommended approach is to add the CA's PEM file to `/usr/local/share/ca-certificates/`,
+and run the `update-ca-certificates` command.
+This will generate the system-managed `/etc/ssl/certs/ca-certificates.crt` list of trusted certificates.
+
+**Arch Linux**
+
+Mount your PEM file into `/etc/ca-certificates/trust-source/anchors/`, then run `update-ca-trust extract` in the container to update the trust store.
+
+**RHEL 8**
+
+Put CA certificates into `/etc/pki/ca-trust/source/anchors/` or `/usr/share/pki/ca-trust-source/anchors/`, and run `update-ca-trust`.
+
+#### Further Reading
+
+- [Our blog post on certificate automation for containers](https://smallstep.com/blog/automate-docker-ssl-tls-certificates/) contains detailed Root Distribution examples that could also provide inspiration for other contexts.
```

Comment:
Nit picking, but if the first bullet ends with a period, should the others as well?

Reply:
I'll fix it in the next PR